tutorials/PytorchCaffe2SuperResolution.ipynb

```diff
@@ -18,9 +18,9 @@
     "executing the model on mobile devices.\n",
     "\n",
     "For this tutorial, you will need to install [onnx](https://github.com/onnx/onnx),\n",
-    "[onnx-caffe2](https://github.com/onnx/onnx-caffe2) and [Caffe2](https://caffe2.ai/).\n",
-    "You can get binary builds of onnx and onnx-caffe2 with\n",
-    "``conda install -c ezyang onnx onnx-caffe2``.\n",
+    "and [Caffe2](https://caffe2.ai/).\n",
+    "You can get binary builds of onnx with\n",
+    "``conda install -c ezyang onnx``.\n",
```

Comment:
here, we should use conda-forge instead of ezyang's channel.

Reply:
Suggest code change?

Reply:
conda install -c ezyang onnx ==> conda install -c conda-forge onnx

A different issue, but it's good to solve it.